data/Debian.yaml

```diff
@@ -0,0 +1,5 @@
+---
+
+openvmtools::desktop_package_name: 'open-vm-toolbox'
+openvmtools::service_name:         'open-vm-tools'
```

Comment:
No need to align these values. It's not normal YAML style.

Reply:
If you insist.